pkg/logger/logger_redis.go

```diff
@@ -0,0 +1,65 @@
+package logger
+
+import (
+	"github.com/gomodule/redigo/redis"
+	"github.com/rs/zerolog/log"
+)
+
+// ContainerLoggerRedis send stdout/stderr to a
+// Redis PubSub channel
+type ContainerLoggerRedis struct {
+	ContainerLogger
+
+	Endpoint      string
+	ChannelStdout string
+	ChannelStderr string
```

Comment:
These property doesn't have to be public
```suggestion
	endpoint      string 
  	channelStdout string
	channelStderr string
```